apps/frontend/src/app/institutions/components/institutions-crud/institution-crud.component.ts

```diff
@@ -172,17 +173,8 @@ export class AppInstitutionCrudComponent implements OnInit {
         this.router.navigate(['/institution/list']);
     }
     verificarFormatoEmail() {
-        let formato = /^[a-zA-Z0-9_.+-]+\@[a-zA-Z0-9-]+(\.[a-z]{2,4})+$/;
-        let email = this.institution.email;
-        if (email) {
-            if (formato.test(email)) {
-                this.disableGuardar = false;
-            } else {
-                this.disableGuardar = true;
-            }
-        } else {
-            this.disableGuardar = false;
-        }
+        let utils = new Utils();
+        utils.verificarFormatoEmail(this.institution.email) ? this.disableGuardar = false : this.disableGuardar = true;
```

Comment:
this.disableGuardar = !utils.verificarFormatoEmail(this.institution.email);

Reply:
dale... lo hago ahi y con el tema de la password también!! me gustó!

Reply:
Listorti